Makefile

```diff
@@ -13,6 +13,10 @@ HTTPS_GIT := https://github.com/cosmos/cosmos-sdk.git
 DOCKER := $(shell which docker)
 PROJECT_NAME = $(shell git remote get-url origin | xargs basename -s .git)
 
+ifeq ($(findstring .,$(VERSION)),)
```

Comment:
When the version contains no dot, then it is not a semantic version. This is the case for a PR branch for example. The CLI requires a valid sematic version so I default to an unused one